pkgs/servers/home-assistant/intents.nix

```diff
@@ -32,13 +34,23 @@ buildPythonPackage rec {
     fetchSubmodules = true;
   };
 
+  patches = [
+    # https://github.com/home-assistant/intents-package/pull/1
+    (fetchpatch {
+      name = "unpin-setuptools-dependency.patch";
+      url = "https://github.com/home-assistant/intents-package/commit/fc03c1ec1001ba3b812f2eea670d1c447cf2acee.patch";
+      hash = "sha256-kTu1+IwDrcdqelyK/vfhxw8MQBis5I1jag7YTytKQhs=";
+    })
+  ];
```

Comment:
Good luck getting anything merged after having mentioned you require this for nixpkgs.